openpathsampling/tests/test_shooting_point_analysis.py

```diff
@@ -197,81 +220,79 @@ def test_non_shooting_steps(self):
         step0 = self.storage.steps[0]
         step1 = self.storage.steps[1]
 
-        assert_equal(self.analyzer.step_key(step0), None)
-        assert_equal(self.analyzer.step_key(step1), None)
+        assert self.analyzer.step_key(step0) is None
+        assert self.analyzer.step_key(step1) is None
 
-        assert_equal(self.analyzer.analyze_single_step(step0), [])
-        assert_equal(self.analyzer.analyze_single_step(step1), [])
+        assert self.analyzer.analyze_single_step(step0) == []
+        assert self.analyzer.analyze_single_step(step1) == []
 
     def test_committor(self):
         committor_A = self.analyzer.committor(self.left)
         committor_B = self.analyzer.committor(self.right)
-        assert_true(len(committor_A) == len(committor_B) == 2)
+        assert len(committor_A) == len(committor_B) == 2
         keys = [self.snap0, self.snap1]
         hashes0 = [self.analyzer.hash_function(k) for k in keys]
         for kA, kB in zip(list(committor_A.keys()), list(committor_B.keys())):
             hashA = self.analyzer.hash_function(kA)
             hashB = self.analyzer.hash_function(kB)
-            assert_equal(hashA, hashB)
-            assert_in(hashA, hashes0)
+            assert hashA == hashB
+            assert hashA in hashes0
             # hash is the same; snapshot is not
         for snap in committor_A:
-            assert_equal(committor_A[snap],
-                         old_div(float(self.analyzer[snap][self.left]), 20.0))
-            assert_almost_equal(committor_A[snap] + committor_B[snap], 1.0)
-            assert_equal(committor_B[snap],
-                         old_div(float(self.analyzer[snap][self.right]), 20.0))
+            assert (committor_A[snap] ==
+                    float(self.analyzer[snap][self.left]) / 20.0)
+            assert pytest.approx(committor_A[snap] + committor_B[snap]) == 1.0
+            assert (committor_B[snap] ==
+                    float(self.analyzer[snap][self.right]) / 20.0)
```

Comment:
I removed `old_div` here as I am pretty sure in python `2.7` `float/float` is identical to the implementation in python `3`

Reply:
Tests were automatically converted from Python 2 to 3. I think the tool to do that automatically replaced `/` with `old_div` in all cases. So yeah, if you see strange use of `old_div` in tests, that's why.

(BTW, on mobile now. Will do a more complete review (much) later.)